app/controllers/Dashboard.scala

```diff
@@ -53,8 +53,13 @@ class Dashboard @Inject()(
     Redirect(config.newDashboardUrl("your-details"))
   }
 
-  def manageProperties() = authenticated { implicit request =>
-    Redirect(config.newDashboardUrl("your-properties"))
+  def manageProperties(clientId: Option[Long] = None, clientName: Option[String] = None) = authenticated {
```

Comment:
it would e nice to remove this file at some point. The dashboard has been live for a very long time now, we should look to move away from these re-directs

Reply:
Good idea but we will create a new story for this and remove, already scope of this story is expanded scope and also this hold up Two stories